en/cocos-service/agc-clouddb.md

```diff
@@ -32,6 +32,7 @@ When using Cloud DB service, if the free resource quota cannot meet the develope
     - Add `getObjectTypeName` and `getPackageName` methods to the query result set object.
     - Remove `setUpgradeProcessMode`, `deleteAll` method.
     - If you are using the `data.zoneId` parameter in the callback, please modify it to `data.zone.zoneId`.
+    - Modify `subscribe` method, supports in using only the equal-value subscription, that is, the `equalTo()` method to construct query conditions.
```

Comment:
Modify `subscribe` method, supports in using only the equal-value subscription, that is, the `equalTo()` method to construct query conditions.
-->
Modify the `subscribe` method, supports using only the equal-value subscription, that is, the `equalTo()` method to construct query conditions.

Reply:
fixed